src/order/rel_classes.lean

```diff
@@ -200,7 +201,17 @@ instance is_order_connected_of_is_strict_total_order'
 @[priority 100] -- see Note [lower instance priority]
 instance is_strict_total_order_of_is_strict_total_order'
   [is_strict_total_order' α r] : is_strict_total_order α r :=
-{..is_strict_weak_order_of_is_order_connected}
+{ }
+
+@[priority 100] -- see Note [lower instance priority]
+instance is_strict_weak_order_of_is_strict_total_order'
+  [is_strict_total_order' α r] : is_strict_weak_order α r :=
+{ ..is_strict_weak_order_of_is_order_connected }
+
+@[priority 100] -- see Note [lower instance priority]
+instance is_strict_weak_order_of_is_strict_total_order
+  [is_strict_total_order α r] : is_strict_weak_order α r :=
+by { haveI : is_strict_total_order' α r := {}, apply_instance }
```

Comment:
Were these instances removed from core?

Reply:
This was a redundant assumption. I've updated the PR description.

Reply:
This was a redundant assumption. I've updated the PR description.